packages/cloud_functions/cloud_functions_platform_interface/lib/src/method_channel_cloud_functions.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of cloud_functions_platform_interface;
+
+class CloudFunctionsException implements Exception {
```

Comment:
let's leave `CloudFunctionsException` in `package:cloud_functions` and not catch the `PlatformException` below. This allows `package:cloud_functions` to be more flexible about handling exceptions (and doesn't limit `CloudFunctionsException` to just the method channel implementation